pkgs/by-name/lo/localsend/package.nix

```diff
@@ -11,27 +11,68 @@
   libayatana-appindicator,
   undmg,
   makeBinaryWrapper,
+  rustPlatform,
+  moreutils,
+  jq,
+  cargo,
+  rustc,
+  fetchzip,
 }:
 
 let
   pname = "localsend";
-  version = "1.15.4";
+  version = "1.16.0";
 
   linux = flutter324.buildFlutterApplication rec {
-    inherit pname;
-    version = "1.15.4-unstable-2024-10-26";
+    inherit pname version;
 
     src = fetchFromGitHub {
       owner = pname;
       repo = pname;
-      rev = "d0e864ddf6ebe1c55188b33f8fdccd534d203c75";
-      hash = "sha256-qaW9LoRQge4K/CAQwbjNxmlUkQmxvaZyBJdAjXbbTw4=";
+      rev = "v${version}";
+      hash = "sha256-0FBHT7hIDVpRVc+a1Y7g35MGzoE66oGf0CvsU1dUqCc=";
     };
 
     sourceRoot = "${src.name}/app";
 
     pubspecLock = lib.importJSON ./pubspec.lock.json;
 
+    customSourceBuilders = {
```

Comment:
Please put this in https://github.com/NixOS/nixpkgs/tree/master/pkgs/development/compilers/dart/package-source-builders, same for hiddify...